readme.md

```diff
@@ -2,9 +2,38 @@
 
 [![CircleCI](https://circleci.com/gh/Actify-Inc/arangodb-net-standard.svg?style=svg)](https://circleci.com/gh/Actify-Inc/arangodb-net-standard)
 
-## Summary
+## Description
 
-`arangodb-net-standard` is a consistent, comprehensive, minimal driver for the [ArangoDB REST API](https://www.arangodb.com/docs/stable/http/). Built using .NET Standard 2.0, the library provides .NET Core and .NET Framework applications with the complete range of features exposed by the ArangoDB REST API.
+A consistent, comprehensive, minimal driver for the [ArangoDB REST API](https://www.arangodb.com/docs/stable/http/). Built using .NET Standard 2.0, the library provides .NET Core and .NET Framework applications with the complete range of features exposed by the ArangoDB REST API.
+
+---
+
+- [arangodb-net-standard: The C#/.NET Standard API driver for ArangoDB.](#arangodb-net-standard--the-c--net-standard-api-driver-for-arangodb)
```

Comment:
Is it not superfluous to include?

Reply:
I guess so.  I used an online TOC generator.  Something has to be at the root of the tree, so I left it as it is.

Reply:
I see, not a big deal anyway so let's leave it like that then.